Sources/ReleaseServer/ReleaseServer.WebApi/Controllers/ReleaseArtifactController.cs

```diff
@@ -106,6 +163,14 @@ public async Task<IActionResult> GetSpecificArtifact([Required] string product,
             return new FileContentResult(response.Payload, contentType);
         }
         
+        /// <summary>
+        /// Retrieves the latest artifact of a specific product.
+        /// </summary>
+        /// <param name="product"></param>
+        /// <param name="os"></param>
+        /// <param name="architecture"></param>
+        /// <response code="200">The specified product exists (the ZIP file with the artifact will be retrieved)</response>
+        /// <response code="500">The product is not available vor the specified platform (OS + arch)</response>
```

Comment:
Same as above: A 500 error code represents a server-side error. 404 (Not Found) would be more appropriate here.

Reply:
fixed

Reply:
see https://github.com/Traeger-GmbH/release-server/pull/50/commits/89c4e62a1637ac6d990d290b01c6e126d5ef6900